CONTRIBUTING.md

```diff
@@ -178,9 +179,23 @@ documentation:
 - For best results use 800 px (width)
 - For best results use 1600 px (height)
 
+#### Images directory
+
+> See the [docs](https://github.com/newrelic/newrelic-quickstarts/blob/main/README.md#getting-started) for more details on `dashboards`
+
+- These images are `optional` and should contain images you want to display within a markdown widget on your Dashboard.
+- File name should be `quickstart_name01`, `quickstart_name02`, etc
+- These images should be stored in the quickstart's images directory. ex: `/quickstart_name01/images`.
+- Must be in `.png`, `.jpg`, `.jpeg` or `.svg` format
+- Each image file must be less than `4MB` in size
+- There should be no more than `6`  dashboard images per dashboard
```

Comment:
cant believe i didnt ask this before 🙈 . if a user includes two dashboards, doesnt this mean they can supply 12 images?

does the code currently account for this? looks like it might be looking for just 6 total for the quickstart.

Reply:
@nr-kkenney good question, I think  we set out to set a 6 dashboard images per dashboard, limit originally but if we are seeing multiple dashboards being addded per pack we may want to consider a higher limit if this accounts for all dashboard images.

Reply:
@nr-kkenney oooh good call yes that would make total sense, i'll alter this to account for the amound of dashboard json files in the folder